Trappist/src/Promact.Trappist.Core/Controllers/CategoryController.cs

```diff
@@ -26,44 +30,47 @@ public CategoryController(ICategoryRepository categoryRepository)
 
         [HttpPost]
         /// <summary>
-        /// Post Method 
-        /// Will Add a Catagory Name in Category Model
+        /// Api to add Category
         ///</summary>
-        /// <param name="category">Object of  class Category</param>
-        /// <returns>object of the class </returns>
-        public IActionResult CatagoryAdd([FromBody] Category category)
+        /// <param name="category">category object contains Category details</param>
+        /// <returns>category objects contains Category details</returns>
```

Comment:
category object will contain only category details. that the name says. no need to write detail explanation.